angular/src/lib/sidebar-nav-item/sidebar-nav-item.component.ts

```diff
@@ -97,6 +100,10 @@ export class SidebarNavItemComponent implements AfterContentInit, OnInit, OnDest
   @Input() isReadOnly: boolean = false;
   /** @prop creates a separator line below the sidebar nav item */
   @Input() separator: boolean = false;
+  /** @prop optional CSS class for the icon left of sidebar nav item title */
+  @Input() iconClass: string = '';
+  /** @prop optional CSS class on the sidebar nav item wrapper div */
+  @Input() navItemClass: string = '';
```

Comment:
I've seen className for the core component in other components...what's the pattern we're following for the main component?

Reply:
switching it to className as its the main component.